.github/dependabot.yml

```diff
@@ -6,7 +6,13 @@
 version: 2
 updates:
   - package-ecosystem: "pip" # See documentation for possible values
-    directory: "/docs/.sphinx" # Location of package manifests
+    directory: "/docs/sphinx" # Location of package manifests
     open-pull-requests-limit: 10
     schedule:
       interval: "daily"
+    labels:
+      - "documentation"
+      - "dependencies"
+      - "ci:no-build"
```

Comment:
This needs to be updated. Aside from that, is everything ready on this PR?

Reply:
Thanks for catching that. Fixed in [7ffaab0](https://github.com/ROCmSoftwarePlatform/hipfort/pull/113/commits/7ffaab08de59add9a0d31c08b1a1d66a21c4ec07)

Yes, this should be ready for review.